assets/app/scripts/services/auth.js

```diff
@@ -276,4 +276,18 @@ angular.module('openshiftConsole')
       }
     }
   };
-}]);
+}])
+
+/* A WebSocket factory for kubernetesContainerTerminal */
+.factory("AuthContainerWebSocket", function(API_CFG, AuthService) {
```

Comment:
I want to consistently use $ws and $http to make sure registered interceptors get called in all cases